src/commands/GetInfoCommand.h

```diff
@@ -57,6 +57,7 @@ class GetInfoCommand : public AudacityCommand
    bool SendClips(const CommandContext & context);
    bool SendEnvelopes(const CommandContext & context);
    bool SendBoxes(const CommandContext & context);
+   bool SendTime(const CommandContext & context);
```

Comment:
Can this be marked as const? 
I am not familiar with code yet but I am just curios.